front/app/containers/Admin/settings/registration/index.tsx

```diff
@@ -52,10 +52,6 @@ const SettingsRegistrationTab = () => {
   // Creating another instance to update follow preferences separately
   const { mutate: updateFollowPreferences } = useUpdateAppConfiguration();
 
-  const userConfirmationIsAllowed = useFeatureFlag({
```

Comment:
`user_confirmation` is allowed for all plans, so this flag does not do anything.